server/datastore/psql/maintenance_request_operations.go

```diff
@@ -1,17 +1,15 @@
 package psql
```

Comment:
any reason why you changed the name of the file to `maintenance_request_operations.go`? instead of just `maintenance_request.go`

Reply:
Did I miss part of a conversation? Why are we renaming this file? `maintenance_requests.go` should be correct, no?

Reply:
yeah it should be `maintenance_requests.go` but in this pr it was changed to `maintenance_request_operations.go`

Reply:
My b. I must have misunderstood something. I'll fix that